src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
 lang: en
 ---
 
-# Ethereum energy consumption {#introduction}
+## Ethereum's energy expenditure {#proof-of-stake-energy}
```

Comment:
This is one of the pages where do want to use an h1 (`#`) to start, since this template does not automatically render the `title` front matter property as an h1.
```suggestion
# Ethereum's energy expenditure {#proof-of-stake-energy}
```

@minimalsm We may want to consider addressing this separately so our markdown pages always consistently render an h1 from the title property.

Reply:
Yeah, we discussed this before but didn't take action. The problem is we have inconsistencies between the title frontmatter and the explicit H1 we're using, so we should probably make those consistent first.

Reply:
@jmcook1186 any reason why we prefer expenditure over consumption here? no hard stance, just curious.

Reply:
no, either is fine by me